src/patterns/page-not-found-pages/index.md.njk

```diff
@@ -55,22 +55,19 @@ Do not use:
 - informal or humourous words like oops
 - red text to warn people
 
-### Service mistake
+### If you do not know what caused the page not found
 
 {{ example({group: "patterns", item: "page-not-found-pages", example: "default", html: true, size: "xl"}) }}
 
-### User mistake
+### If you know the page not found is because of a broken link or button inside the service
 
-{{ example({group: "patterns", item: "page-not-found-pages", example: "user-mistake", html: true, size: "l"}) }}
+{{ example({group: "patterns", item: "page-not-found-pages", example: "inside-service", html: true, size: "xl"}) }}
 
-### If you do not know why the page is not found
+### If the page exists but is no longer available
```

Comment:
I suggest removing this example and the associated guidance, and consider adding it as guidance to Confirmation pages and Question pages.

I don't think this is technically one of the 400 pages.

Additionally, I think if a user has bookmarked a confirmation page or a page in the middle of a flow, there is probably a more helpful interstitial page that we could show them than a Page not found.

Reply:
Agreed. I am not sure what we would say to someone that would not end up as technical information, but adding something to question pages and confirmation pages is a good idea.